atlas_splitter/layer_splitter/isocortex_layer_23.py

```diff
@@ -127,23 +125,24 @@ def edit_hierarchy(
             # Create children
             new_children = []
             for layer in ["layer_2", "layer_3"]:
+                new_child = copy.deepcopy(child)
+                new_child["acronym"] = child["acronym"][:-3] + layer[-1]
```

Comment:
How does this work for layers that are not ending to integers?

Reply:
I don't know, I did not change this:
```
- new_child["acronym"] = child["acronym"][:-3]
- new_child["acronym"] = new_child["acronym"] + layer[-1]
+ new_child["acronym"] = child["acronym"][:-3] + layer[-1]
```

Reply:
I assume given that only 2/3 layer is mentioned in this module, that this is the case for the splitter. So we are good.

Reply:
I missed the line just above `if child["acronym"].endswith("2/3"):`
It's good indeed.